docs/containers-registries.conf.5.md

```diff
@@ -109,6 +109,18 @@ internet without having to change `Dockerfile`s, or to add redundancy).
 *Note*: Redirection and mirrors are currently processed only when reading images, not when pushing
 to a registry; that may change in the future.
 
+#### Normalization of docker.io references
+
+The Docker Hub `docker.io` is handled in a special way, because every push and
+pull operation gets internally normalized with `/library` if no other specific
+namespace is defined (for example on `docker.io/username/image`).
+
+This means that a pull of `docker.io/alpine` will be internally translated to
+`docker.io/library/alpine`. A pull of `docker.io/user/alpine` will not be
+rewritten because this is already the correct remote path.
+
+Note that the upper normalization happens to match the behavior of Docker.
```

Comment:
s/the upper normalization/the above-described normalization/ ?

Possibly move it to the first paragraph in a parenthesis.

Reply:
Fixed :+1: